test/Elastic.Apm.AspNetCore.Tests/DistributedTracingAspNetCoreTests.cs

```diff
@@ -38,9 +38,20 @@ public Task InitializeAsync()
 					{
 						Startup.ConfigureServicesExceptMvc(services);
 
-						services.AddMvc()
-							.AddApplicationPart(Assembly.Load(new AssemblyName(nameof(SampleAspNetCoreApp))))
-							.SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+						services
+#if NETCOREAPP3_0
+							.AddMvc()
+							.SetCompatibilityVersion(CompatibilityVersion.Version_3_0)
```

Comment:
@gregkalapos, As I remember we discussed it and aligned on deleting `SetCompatibilityVersion`.

Reply:
Thanks! Yes, these seem to be leftovers - removed them.